bots/tests-invoke

```diff
@@ -166,9 +166,9 @@ class PullTask(object):
             traceback.print_exc()
             return "Rebase failed"
 
-        # COMPAT: If the bots directory doesn't exist in this branch, check it out from master
+        # Always use bots/ and test images from master
         try:
-            if subprocess.call([ "git", "ls-tree", "-d", "HEAD:bots/"], stdout=DEVNULL) != 0:
+            if self.base != "master":
```

Comment:
Good point, fixed. I just added a nonempty check, which is easier to understand and follow in logs.